RecoBTag/Configuration/python/RecoBTag_EventContent_cff.py

```diff
@@ -76,8 +81,10 @@
         'keep *_pfCombinedInclusiveSecondaryVertexV2BJetTags_*_*',
         'keep *_pfCombinedMVABJetTags_*_*',
         'keep *_pfCombinedMVAV2BJetTags_*_*',
-	'keep *_pfCombinedSecondaryVertexSoftLeptonBJetTags_*_*',
-##	'keep *_pfCombinedSecondaryVertexSoftLeptonCtagLJetTags_*_*',
-        'keep *_inclusiveCandidateSecondaryVertices_*_*'
+        'keep *_pfCombinedSecondaryVertexSoftLeptonBJetTags_*_*',
+        'keep *_inclusiveCandidateSecondaryVertices_*_*',
```

Comment:
@mverzett, to keep c-tag IVF vertices, it should be sufficient to add `'keep *_inclusiveCandidateSecondaryVerticesCvsL_*_*'` after `'keep *_inclusiveCandidateSecondaryVertices_*_*'` (in 3 separate places in the cff file).